src/common/balances.lua

```diff
@@ -7,9 +7,10 @@ local balances = {}
 
 --- Transfers the ANT to a specified wallet.
 ---@param to string - The wallet address to transfer the balance to.
+---@param allowUnsafeAddresses boolean Whether to allow unsafe addresses
```

Comment:
nit: can share type declarations and desriptions e.g:
 ```lua
 @alias AllowUnsafeAddresses boolean Whether to allow unsafe addresses
 
 
 @param allowUnsafeAddresses AllowUnsafeAddresses
 ```

Reply:
[2242e41](https://github.com/ar-io/ar-io-ant-process/pull/38/commits/2242e41d9b0e15fb533569c5c493dc2a4042a6b1)